code/modules/mob/living/simple_animal/hostile/skeleton.dm

```diff
@@ -85,11 +85,9 @@
 /mob/living/simple_animal/hostile/skeleton/revenant
 	name = "haunted remains"
 	desc = "It's a skeleton alright, but it seems to be fuelled by haunted energies."
+	icon_state = "revskeleton"
+	icon_living = "revskeleton"
 	gold_core_spawnable = NO_SPAWN
 	maxHealth = 70
 	health = 70
-	var/lastthingtheysaid = "Kill..."
-
-/mob/living/simple_animal/hostile/skeleton/revenant/Life()
-	if(prob(10))
-		say("[lastthingtheysaid]")
\ No newline at end of file
+	speak = list("")
```

Comment:
Since i think that speak is already list() in simple_animal this should be unnecessary, additionally if you do list("") it will count "" as a list element and you'll get 50% chance of the skeleton saying nothing

Reply:
XDTM is correct